### DISCUSSION TITLE
今天遇到了一个奇怪的人

### DISCUSSION THREAD

**Discussion by 132641515Zzy** [share121/inter-knot#458] [其他]
![image](https://github.com/user-attachments/assets/d148ad6e-344b-40e3-8804-48f23d3b4056)
卡牌店开张没多久我家孙子就带朋友来参观，但是后面来了个小子要买我的羁绊之卡，我拒绝之后还让我走着瞧，真是的，一天的好心情没了
另外欢迎大家光顾我家卡牌店，现在来买卡牌会有优惠哦

**Comment by kano0x0** [comment]
你好，我想预定一张黑魔导少女，地址是五分街3001号2号公寓

**Reply by 132641515Zzy** [reply]
黑魔导系列可是稀有卡啊，现在也只有我孙子持有一张，你还是老实的开卡包吧
